modules/ROOT/partials/configuration/link_default_protocol.adoc

```diff
@@ -11,11 +12,16 @@ Default Value: `+'https'+`
 
 === Example: Using `+link_default_protocol+`
 
-[source,js]
+:plugin_list: link
+ifeval::["{plugincode}" == "autolink"]
+:plugin_list: link autolink
+endif::[]
```

Comment:
Can you put this at the top please, as attributes defined in the middle like this are hard to see and make it look like content.

- [ ] Move to the top of the file

Reply:
Done